backend/migrate_encrypt_tokens.py

```
@@ -0,0 +1,115 @@
+    Migrate all users' git tokens from plain text to encrypted format
+
+    Args:
+        db: Database session
+
+    Returns:
+        Number of users updated
+    """
+    updated_count = 0
+
+    # Get all users
+    users = db.query(User).filter(User.is_active == True).all()
```

Ruff (0.14.4)

Avoid equality comparisons to `True`; use `User.is_active:` for truth checks

Replace with `User.is_active`

(E712)